evm/src/fuzz/strategies/state.rs

```diff
@@ -87,6 +87,17 @@ pub fn build_initial_state<DB: DatabaseRef>(db: &CacheDB<DB>) -> EvmFuzzState {
         // Insert basic account information
         state.insert(H256::from(*address).into());
 
+        // Insert push bytes
+        if let Some(code) = &account.info.code {
+            if !state.cache.contains(address) {
+                state.cache.insert(*address);
```

Comment:
can do this in one step

```suggestion
            if state.cache.insert(*address) {
```